namespace-provisioner/how-tos.hbs.md

```diff
@@ -0,0 +1,254 @@
+# Namespace Provisioner How-to Guide
+
+This page provides guidance for more advanced use-cases associated with Namespace Provisioner.
+
+The available guides are as follows:
+
+* [Data values templating](#data-values-templating)
+* [GitOps customizations](#gitops-customizations):
+   1. [Extending the default resources that get provisioned](#extending-default-resources)
+   2. [Customizing OOTB default resources](#customizing-default-resources)
+   3. [Control reconcile behavior of namespace provisioner for certain resources](#control-reconcile-behavior)
+   4. [Control the desired-namespace ConfigMap via GitOps](#control-desired-namespaces)
+
+## <a id="data-values-templating"></a>Data values templating guide
+
+Customize your custom resources with data values from TAP values and data from `desired-namespaces` ConfigMap.
+
+Namespace provisioner inherits all of the configuration in the [`desired-namespaces`](about.hbs.md#nsp-component-desired-namespaces-configmap) as well as the tap values under the key `tap_values` making it available for Platform Operators to use as ytt `data.values` when [extending the resources via GitOps](#extending-default-resources).
```

Comment:
Should this say:

Namespace Provisioner inherits all of the configuration in the [`desired-namespaces Config map`](about.hbs.md#nsp-component-desired-namespaces-configmap) as well as any additional configuration under 'additional sources' in tap-`tap_values.yaml`.